packages/connector-jsdom/package.json

```diff
@@ -9,8 +9,10 @@
   "dependencies": {
     "@hint/utils-connector-tools": "^1.0.3",
     "canvas-prebuilt": "^1.6.5-prerelease.1",
-    "jsdom": "^11.12.0",
-    "node-pre-gyp": "^0.11.0"
+    "jsdom": "git://github.com/sarvaje/jsdom.git#v12-fixes",
```

Comment:
Is there an PR for this? If so can you link it here. Thanks!

Reply:
The problem is that I have 2 PR for this :(

Reply:
Can you link to them here. Thanks!